sass/components/_breadcrumbs.scss

```diff
@@ -0,0 +1,9 @@
+.breadcrumbs li::after {
```

Comment:
that said, i think this is a perfectly fine example of where new css could be appropriate